osbs/build/build_request.py

```diff
@@ -890,9 +922,13 @@ def render_tag_from_config(self):
         unique_tag = self.spec.image_tag.value.split(':')[-1]
         tag_suffixes = {'unique': [unique_tag], 'primary': []}
 
-        if self.spec.build_type.value == BUILD_TYPE_ORCHESTRATOR and not self.scratch:
+        # These changes are just temporary
+        # TODO: revert this code block to run on orchestrator
+        if self.spec.build_type.value == BUILD_TYPE_WORKER and not (self.scratch or self.isolated):
```

Comment:
I'd like these if conditionals to be a little bit cleaner to read.
Something like this:

```python
if self.spec.build_type.value == BUILD_TYPE_WORKER:  # Will be BUILD_TYPE_ORCHESTRATOR once workaround is  not needed
  if self.scratch:
    # No additional tags for scratch builds
    pass 
  elif self.isolated:
    tag_suffixes['primary'].extend(['{version}-{release}'])
  else:
    tag_suffixes['primary'].extend(['latest', '{version}', '{version}-{release}'])
    tag_suffixes['primary'].extend(self._repo_info.additional_tags.tags)
```